src/tests/src/libs/antares/study/CMakeLists.txt

```diff
@@ -6,3 +6,13 @@ add_subdirectory(output-folder)
 add_subdirectory(short-term-storage-input)
 add_subdirectory(thermal-price-definition)
 add_subdirectory(constraint)
+
```

Comment:
Question : why not make a new test directory for testing basic study functionalities ?

Reply:
I just followed the structure of the libs themselves: the study class is at the root dir of the "study" lib.
I thought it was the implicit rule, but actually I see that it's not the case for STS and thermal prices.

Reply:
Yes, sorry, I wanted to remove this comment, but I forgot